perfplot/main.py

```diff
@@ -212,7 +213,7 @@ def show(*args, **kwargs):
     return
 
 
-def save(filename, *args, **kwargs):
+def save(filename, transparent=False, *args, **kwargs):
```

Comment:
Default `True` please. (Let's not change the behavior.)

Reply:
Sorry, my mistake.